### DISCUSSION TITLE
Bohm Dialog

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#225] [General]
Write up about bohm dialog. research experiences with it.

**Comment by baouroux** [comment]
Would love to follow up the thread here....I'm personnaly putting together to proposal to bring narratives and art into the Interfaith dialog space, inspired by Bohmian dialogue: https://www.tyler.world/interfaith-narratives-and-art-proposal

I came across this paper built on experience of various groups running Bohmian dialogue gatherings: "The Ethics of Presence- New Paths in Interfaith Dialogue" 
- https://www.academia.edu/21467066/The_Ethics_of_Presence_New_Paths_in_Interfaith_Dialogue

Olen Gunnlaugson in Canada has been working a lot in the application of both the Integral, developmental and Bohmian dialogue / We Space convening....
One of his books (I have yet to read) is here: [Cohering the Integral We Space Engaging Collective Emergence, Wisdom and Healing in Groups](https://www.amazon.ca/Cohering-Integral-Space-Collective-Emergence/dp/0986282634)

I am wondering if my 'witnessing' portraits, filmed from silence, might evolve into a possible branch of 'ansynchronous' dialogue with Bohmian principles....perhaps a stretch, but hoping to explore further with greater numbers of portraits coming together!
